content/rs/security/encryption/_index.md

```diff
@@ -0,0 +1,70 @@
+---
+Title: Encryption in Redis Enterprise Software
+linkTitle: Encryption
+description: Encryption in Redis Enterprise Software.
+weight: 60
+alwaysopen: false
+toc: "true"
+categories: ["RS"]
+aliases: 
+---
+
+Redis Enterprise Software uses encryption to secure communications between clusters, nodes, databases, and clients and protect [data in transit](https://en.wikipedia.org/wiki/Data_in_transit), [at rest](https://en.wikipedia.org/wiki/Data_at_rest), and [in use](https://en.wikipedia.org/wiki/Data_in_use).
```

Comment:
```suggestion
Redis Enterprise Software uses encryption to secure communications between clusters, nodes, databases, and clients and protects [data in transit](https://en.wikipedia.org/wiki/Data_in_transit), [at rest](https://en.wikipedia.org/wiki/Data_at_rest), and [in use](https://en.wikipedia.org/wiki/Data_in_use).
```

Reply:
You can either change "protect" to "protects" or "to protect"

Reply:
Fixed